cep-deprecation.md

```diff
@@ -0,0 +1,155 @@
+<table>
+<tr><td> Title </td><td> Conda Deprecation Schedule </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Ken Odegard &lt;kodegard@anaconda.com&gt; </td></tr>
+<tr><td> Created </td><td> May 20, 2022 </td></tr>
+<tr><td> Updated </td><td> July 20, 2022 </td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Describes a deprecation schedule to properly warn about upcoming removals from the codebase. This policy expands on ideas and terminology defined in the Conda Release Schedule (CEP-?).
```

Comment:
```suggestion
This CEP describes a deprecation schedule to properly warn about upcoming removals from the codebase. This policy expands on ideas and terminology defined in the Conda Release Schedule (CEP-?).
```

Reply:
☝️ suggestion gets rid of the sentence fragment, but I'm also curious which codebase this CEP applies to? Just the `conda` codebase? Or all of our projects?

Reply:
For now, the deprecation policy will only apply to projects that have adopted the bi-monthly release schedule. I'll update accordingly.